wlinux-setup.d/docker.sh

```diff
@@ -159,6 +159,27 @@ if (whiptail --title "DOCKER" --yesno "Would you like to install the bridge to D
             fi
 
         fi
+
+        cat << 'EOF' >> create-mnt-c-link
+#!/bin/bash
+
+if [[ -z $(ls -A /mnt/c) ]]; then
+
+    rm -d /mnt/c #Ensure that we only delete the directory if it is empty
+    ln -s $(wslpath -u "C:\\") /mnt/c
+fi
+EOF
+        sudo cp create-mnt-c-link /usr/bin/create-mnt-c-link
```

Comment:
Should we give the users some warning here about why they are being asked to sudo?

Reply:
Ups... the code should be inside this `if (whiptail --title "DOCKER" ...` so the user has already approved sudo for changing the root. I will fix it